src/foamadapter/framework/context.py

```diff
@@ -0,0 +1,24 @@
+from typing import Annotated, Any, TypeVar
+
+from pydantic import BaseModel
+
+T = TypeVar("T")
+Model = Annotated[T, "models"]
+Field = Annotated[T, "fields"]
+
+
+class FieldUpdates(dict):
```

Comment:
This seems to be a quite central class, but i dont understand what it is supposed to represent, what does FieldUpdates mean? Does it hold the data after a step  or an operation was performed? Would it be `UpdatedSolverState`?